tasks/main.yml

```diff
@@ -67,8 +52,30 @@
 - name: load config from file
   ansible.builtin.shell: "{{ ldaptoolbox_openldap_slapd_cli_cmd }} restoreconfig"
 
+- name: build generated data from template
+  ansible.builtin.shell: "{{ ldaptoolbox_openldap_slapd_cli_cmd }} importdatatemplate"
+  when: ldaptoolbox_openldap_generate_data_ldif is defined
```

Comment:
I have kept this, but I really don't see the point of automating the import of sample data